atlp-rwanda/rca-barefoot-team6-bn
# Chore gh actions 184638363

#### What does this PR do?

This PR serves to integrate Node.js CI/CD github actions workflows.

#### Description of  the task to be completed?

Has the following functionalities:

-  Setup repo CI steps.
- Setup repo CD step.
- Setup readme status badge.

#### How should this be manually tested?

1. After making a PR or merging the `develop` branch, you can check the [Actions](https://github.com/atlp-rwanda/rca-barefoot-team6-bn/actions) page on the github web to see how the workflows are running. 

2. You can also check the [readme](https://github.com/atlp-rwanda/rca-barefoot-team6-bn/blob/develop/README.md) file to see a realtime status for the jobs. 


#### Any background context you want to provide?

The jobs are currently passing since they haven't been any codes written, no tests implemented, and no deployment environments configured yet. Once, these are ready, the workflows will be evident.

#### What are the relevant pivotal tracker stories?

[#184638363](https://www.pivotaltracker.com/story/show/184638363)

#### Screenshots (if appropriate)

---
